dev/com.ibm.ws.org.apache.cxf.cxf.rt.frontend.jaxws.3.2/src/org/apache/cxf/jaxws/ServiceImpl.java

```diff
@@ -247,20 +280,34 @@ private List<WebServiceFeature> getAllFeatures(WebServiceFeature[] features) {
 
     private JaxWsClientEndpointImpl getJaxwsEndpoint(QName portName, AbstractServiceFactoryBean sf,
                                       WebServiceFeature...features) {
+        
+        boolean isFinestEnabled = LOG.isLoggable(Level.FINEST);   // Liberty Change issue #26529
         Service service = sf.getService();
         EndpointInfo ei;
         if (portName == null) {
             ei = service.getServiceInfos().get(0).getEndpoints().iterator().next();
+            if (isFinestEnabled) {
+                LOG.finest("Jaxws endpoint that is obtained from service > ServiceInfo(0) since portname is null: " + ei);  // Liberty Change issue #26529
```

Comment:
```suggestion
                LOG.finest("EndpointInfo obtained from ServiceInfo because portname is null: " + ei);  // Liberty Change issue #26529
```